sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -819,42 +819,9 @@ object SparkSession extends Logging {
      * Sets a config option. Options set using this method are automatically propagated to
      * both `SparkConf` and SparkSession's own configuration.
      *
-     * @since 2.0.0
-     */
-    def config(key: String, value: String): Builder = synchronized {
```

Comment:
This breaks binary compatibility so we can't simply do this.

Reply:
@HyukjinKwon 
thx to comment!

so it is possible..?

```scala
    def conf(key: String, value: Any): Builder = synchronized {
      options += key -> value.toString
      this
    }
```

Reply:
@HyukjinKwon 
I am done.

if you have a free time, please check!
don't want this code. please close pr.
thx.